CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 - Add sentence id field (`sent_id`) to generated sentence corpora
 - Processes TEI/XML documents to yield separate chunks for body text and footnotes, with each footnote yielded individually as a separate element
+- Add a method for generating sentence embeddings from sentence corpus
```

Comment:
Suggesting an edit since the method now doesn't take a corpus file directly
```suggestion
- Add a method for generating sentence embeddings from a list of sentences
```